### ISSUE TITLE
Increase the accessibility of forms

### ISSUE THREAD

**Issue by mamta** [DominoKit/domino-ui#368]
Hi, to increase the accessibility the input and label elements should be linked using for and id attributes. We connect all the elements using -

 public static void setIdAndForAttribute(HTMLElement inputElement, HTMLElement labelElement) {
        String uuid = UUID.get();
        labelElement.setAttribute("for", uuid);
        inputElement.setAttribute("id", uuid);
    }
    
It is nice if it comes from the domino-ui. What do you think?

Thanks and regards.

**Comment by mamta** [comment]
great, thanks.
